aspnetcore/includes/csharp-8-required.md

```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> The sample below requires C# 8.0 or later.
```

Comment:
@scottaddie I added csharp-8-required.md to the includes directory. This wasn't here when you first approved the PR. Does this look OK to you? Thanks.